internal/callbacker/callbacker_api/callbacker_api.proto

```diff
@@ -0,0 +1,55 @@
+syntax = "proto3";
+
+option go_package = ".;callbacker_api";
+
+package callbacker_api;
+
+import "google/protobuf/timestamp.proto";
+import "google/protobuf/empty.proto";
+
+service CallbackerAPI{
+  rpc Health (google.protobuf.Empty) returns (HealthResponse) {}
+  rpc SendCallback (SendCallbackRequest) returns (google.protobuf.Empty) {}
+}
+
+// Note: Values of the statuses have a difference between them in case
+// it's necessary to add another values in between. This will allow to
+// create new statuses without changing the values in the existing db.
+enum Status {
+  UNKNOWN = 0;
+  QUEUED = 10;
+  RECEIVED = 20;
+  STORED = 30;
+  ANNOUNCED_TO_NETWORK = 40;
+  REQUESTED_BY_NETWORK = 50;
+  SENT_TO_NETWORK = 60;
+  ACCEPTED_BY_NETWORK = 70;
+  SEEN_IN_ORPHAN_MEMPOOL = 80;
+  SEEN_ON_NETWORK = 90;
+  DOUBLE_SPEND_ATTEMPTED = 100;
+  REJECTED = 110;
+  MINED = 120;
+}
+
+// swagger:model HealthResponse
+message HealthResponse {
+  google.protobuf.Timestamp timestamp = 1;
+}
+
+// swagger:model SendCallbackRequest
+message SendCallbackRequest {
+  repeated CallbackEndpoint callback_endpoints = 1;
+  string txid = 2;
+  Status status = 3;
```

Comment:
Could we use here just an integer instead?

I think `callbacker` probably doesn't need to know any logic about statuses anyway

Reply:
I see now, it's used here: https://github.com/bitcoin-sv/arc/blob/93d51c633c3983b586c392391355bc18cf28dd5e/internal/callbacker/callbacker.go#L179

Nevermined